review

```diff
@@ -98,12 +62,13 @@ createRevBranch() {
     else
         put "Branches for this review already exist. New number is assigned."
         lastBranchNumber=$(echo $2 | $SED_COMMAND 's/^.*_\([0-9]\+\)$/\1/g')
-        nextBranchNumber=$(( lastBranchNumber + 1))
+        nextBranchNumber=$(( lastBranchNumber + 1 ))
         revBranch="$1_$nextBranchNumber"
     fi
 
     put "Creating new branch for review $revBranch"
     exe git branch $revBranch
+    echo $revBranch
```

Comment:
Script should not echo various lines that are not user messages.

Reply:
Errrr.. what do you mean exactly? This is how you return strings in bash, isn't it? 😅
(And by that I mean that nothing is actually printed on the console, it's simply returned consumer of the function)